woocommerce/woocommerce-android
# Feature/store creation category step UI

Closes: #8008


### Description
Adds UI for new step in store creation flow: Store category. Expected design: 

![Screenshot 2022-12-09 at 17 18 18](https://user-images.githubusercontent.com/2663464/206745710-a434ced4-1794-449e-b283-11d447cffd59.png)

Couple things to keep in mind

- Displayed categories right now are hardcoded categories used as test data
- Added a feature flag to only trigger this step in debug mode
- There is a known UI glitch on dark mode. I'm waiting for input on what colors to use for selected mode in dark mode. 

### Testing instructions
Start native store creation flow
Navigate to store name step, enter a name and click continue
Land on store category step: 
- Check store name is displayed on top
- Check continue button is disabled until some category is selected
- Check "Skip" button navigates to domain selection step whether you select a category or not. 

### Images/gif


https://user-images.githubusercontent.com/2663464/206746601-f6d9a414-e25f-497d-a462-f549298f0ab0.mp4
